src/helpers/buildWalletSections.tsx

```diff
@@ -133,8 +126,8 @@ const withBriefBalanceSection = (
     {
       type: 'PROFILE_STICKY_HEADER',
       uid: 'assets-profile-header-compact',
-      value: totalValue,
-      isLoadingUserAssets,
+      value: accountBalance,
+      isLoadingBalance,
```

Comment:
ok I think there's an existing issue here - `PROFILE_STICKY_HEADER` is the one that is getting passed these props but the actual `ProfileStickyHeader` component doesn't use them. 
Instead, the `PROFILE_BALANCE_ROW` uses them but was only getting passed the `accountBalance` / `totalValue` and not the `isLoadingBalance` 

I think we need to:
1. update the types for `PROFILE_STICKY_HEADER` in ViewTypes
2. update the types for `PROFILE_BALANCE_ROW` in ViewTypes
3. pass along `isLoadingBalance` for `PROFILE_BALANCE_ROW` in this file
4. remove `value` and `isLoadingBalance` from the `PROFILE_STICKY_HEADER` in this file

Reply:
for #5, now the PROFILE_BALANCE_ROW only shows if `!hasTokens && !isLoadingUserAssets && !isLoadingBalance` is false. so if the user has no tokens, user's assets aren't being loaded, and user's balance isn't being loaded, then don't show the balance row.